### PR TITLE
.net sdk and orleans versions set up correctly

### PR DESCRIPTION
**Pull request by denisivan0v**
@xakep139 please have a look

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.302"
```
Comment (xakep139) [inline global.json:3]: 2.1.506 is the latest 2.1 LTS version

Comment (denisivan0v) [reply global.json:3]: That's correct, but it is not installed at AppVeyor at the moment. So I've decided to use old good version :)

Comment (denisivan0v) [reply global.json:3]: See https://ci.appveyor.com/project/DenisIvanov/orleans-persistence-cassandra/build/job/ndl81gfruia9dor8

**Review by xakep139 — APPROVED** [review]

**Comment by xakep139** [comment]
LGTM

**Pull request merged**